File: API/XApp6/XApp6/XApp6/Models/Curso.cs
using System;
namespace XApp6.Models
{
		public class Curso
		{
			public int id { get; set; }
			public string nome { get; set; }
            public string tipodecurso { get; set; }
            public string responsavel { get; set; }
            public string descricao { get; set; }
	        public int telefone { get; set; }
	        public List<Disciplinas> disciplinas { get; set; }
	        public List<Evento> eventos { get; set; }
	        public List<Documento> documentos { get; set; }
            public List<Utilizador> utilizadores { get; set; }
            public List<Propina> propinas { get; set; }
		}
    }

File: finalbackoffice/backoffice/app/Curso.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;


class Curso extends Model
{
	protected $table = "cursos";
	protected $primaryKey = "id";
	protected $fillable = array("nome", "tipodecurso", "responsavel", "descricao", "email", "telefone");
	public $timestamps = true;
	
	public function disciplinas()
	{
		return $this->belongsToMany('App\Disciplina', 'curso_disciplina')->withTimestamps();
	}
	
	public function documentos()
	{
		return $this->belongsToMany('App\Documento', 'curso_documento')->withTimestamps();
	}
	public function eventos()
	{
		return $this->belongsToMany('App\Evento', 'curso_evento')->withTimestamps();
	}
	public function propinas()
	{
		return $this->belongsToMany('App\Propina', 'curso_propina')->withTimestamps();
	}
	
	public function utilizadores()
	{
		return $this->belongsToMany('App\Utilizador', 'curso_utilizador')->withTimestamps();
	}
	
}
